app/src/main/java/de/danoeh/antennapod/fragment/EpisodeFragment.java

```diff
@@ -0,0 +1,182 @@
+package de.danoeh.antennapod.fragment;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.viewpager2.adapter.FragmentStateAdapter;
+import androidx.viewpager2.widget.ViewPager2;
+
+import com.google.android.material.tabs.TabLayout;
+import com.google.android.material.tabs.TabLayoutMediator;
+
+import org.greenrobot.eventbus.EventBus;
+
+import java.util.ArrayList;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.activity.MainActivity;
+import de.danoeh.antennapod.core.util.ChapterUtils;
+import de.danoeh.antennapod.core.util.playback.Playable;
+import de.danoeh.antennapod.core.util.playback.PlaybackController;
+import io.reactivex.Maybe;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+
+/**
+ * Hosts Cover and Description Fragment
+ */
+public class EpisodeFragment extends Fragment {
```

Comment:
It doesnt really make a difference if wrapper or not, i think the tabs make sense here for transcripts, so i would like to leave it like this i guess? But transcripts will be rare so i dont know, but you said there might be other tabs as well?

Reply:
I initially thought there would be description, transcript and user notes. After thinking about it for some time, I think that the user notes should be displayed as an item in the toolbar because we need to access them for the non-playing episodes, too. Then it can be consistent. Not sure how prominent we want the transcripts to be. Maybe they can be displayed behind a button or in the toolbar.

I think **not** having a WebView in a ViewPager in another ViewPager in a BottomSheet could also make it more easy to handle scrolling.

Reply:
Yea I don't know, I think the scroll is working fine now with a new fix for the 45° issue, I'll leave it for now, maybe we can think again before merging maybe then we'll know if there's a need for the Transcript fragment, because for the Transcript I think the tablayout would be a good solution. If not we can remove the tablayout&pager alltogether.

Reply:
removed tablayout & viewpager